po/noVNC.pot

```diff
@@ -6,9 +6,9 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: noVNC 1.1.0\n"
+"Project-Id-Version: noVNC 1.0.0\n"
```

Comment:
It should be 1.1.0 since that's the upcoming release.

Reply:
OK